src/File.roc

```diff
@@ -51,27 +56,28 @@ writeBytes = \path, bytes ->
 ## ```
 ## This opens the file first and closes it after writing to it.
 ##
-## To write unformatted bytes to a file, you can use [File.writeBytes] instead.
+## > Note: to write unformatted bytes to a file, you can use [File.writeBytes] instead.
 writeUtf8 : Path, Str -> Task {} [FileWriteErr Path WriteErr]
 writeUtf8 = \path, str ->
     toWriteTask path \bytes -> Effect.fileWriteUtf8 bytes str
 
-## Deletes a file from the filesystem.
+## Deletes a file from the filesystem. Performs a [`DeleteFile`](https://docs.microsoft.com/en-us/windows/win32/api/winbase/nf-winbase-deletefile)
```

Comment:
Personally I like to have docs start with a simple sentence (or maaaaybe two) that explains what the function does at a high level, and then include details later on. This way, if I'm reading the docs for the first time, I can start by getting a basic idea of what they do (e.g. "does this solve my problem?") and then see an example to understand how it's used in practice, and then once I've decided "okay yeah this does seem like what I want" then I keep reading to learn relevant details.

That was the reasoning behind explaining the OS-specific function calls later.

Reply:
Rgr, that makes sense. I'll make some more updates.

Reply:
Did another pass and updated all of the doc comments to this style.